VideoLocker/res/layout/discussion_user_profile_row.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/discussion_user_profile_row"
```

Comment:
Yeah, but since it is being used consistently, there's no harm in having it that way.